.travis.yml

```diff
@@ -11,16 +15,21 @@ node_js:
 
 before_install:
  - git submodule update --init --recursive
+ - npm install -g grunt-cli
 
 script:
  - mocha
  - ./node_modules/jscs/bin/jscs main.js lib/ test/
  - ./node_modules/jscs/bin/jscs main.js main.js lib/ test/
+ - grunt build
+ - sh -c 'if [ "$TRAVIS_NODE_VERSION" = "0.10" ]; then grunt test-browser; fi';
```

Comment:
With grunt-saucelabs, we don't have to use Travis-CI's addon; this way we avoid opening the tunnel to saucelabs for every javascript version.